test/TesterInternal/MembershipTests/AzureMembershipTableTests.cs

```diff
@@ -11,15 +13,13 @@ namespace UnitTests.MembershipTests
     /// <summary>
     /// Tests for operation of Orleans Membership Table using AzureStore - Requires access to external Azure storage
     /// </summary>
-    public class AzureMembershipTableTests : MembershipTableTestsBase
+    public class AzureMembershipTableTests : MembershipTableTestsBase, IClassFixture<AzureStorageBasicTestFixture>
     {
         public AzureMembershipTableTests()
         {
             TraceLogger.AddTraceLevelOverride("AzureTableDataManager", Severity.Verbose3);
             TraceLogger.AddTraceLevelOverride("OrleansSiloInstanceManager", Severity.Verbose3);
             TraceLogger.AddTraceLevelOverride("Storage", Severity.Verbose3);
-
-            TestUtils.CheckForAzureStorage();
```

Comment:
Because these tests stopped working in the move to xunit. This needs to run before all other tests in the class. so I fixes it like [this](https://github.com/shayhatsor/orleans/blob/RefactorSql/test/TesterInternal/MembershipTests/AzureMembershipTableTests.cs#L16)